core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -51,8 +51,16 @@ package object config {
   private[spark] val DRIVER_CLASS_PATH =
     ConfigBuilder(SparkLauncher.DRIVER_EXTRA_CLASSPATH).stringConf.createOptional
 
+  private[spark] val DRIVER_DEFAULT_JAVA_OPTIONS =
```

Comment:
Not used?

Reply:
Thought would be good to have it because users can see the structure (string, optional, etc) but I've removed it.

Reply:
Well, looking at it again my original thought is bad because the type of the parameter is given where it's prepended.